linkerd/docs/routers.md

```diff
@@ -186,9 +178,10 @@ Key | Default Value | Description
 hostConnectionPool | An empty object | see [hostConnectionPool](#host-connection-pool).
 tls | no tls | The router will make requests using TLS if this parameter is provided.  It must be a [client TLS](#client-tls) object.
 loadBalancer | [p2c](#power-of-two-choices-least-loaded) | A [load balancer](#load-balancer) object.
-retries | see [retries](#retries) | A [retry policy](#retries) for all clients created by this router.
+failFast | `false` | If `true`, connection failures are punished more aggressively. Should not be used with small destination pools.
+requeueBudget | see [retries](#retries) | A [requeue budget](#retries) for connection-level retries.
 failureAccrual | 5 consecutive failures | a [failure accrual policy](#failure-accrual) for all clients created by this router.
-timeoutMs | no timeout | Per-request timeout in milliseconds.
+requestAttemptTimeoutMs | no timeout | The timeout, in milliseconds, for each attempt (original or retry) of the request made by this client.
```

Comment:
Oh I see what you mean.  Retries are above timeouts in the client stack.  So timeouts don't get requeued but if a requeue happens, the requeue gets a new timeout.